include/net/fota_download.h

```diff
@@ -139,8 +139,7 @@ int fota_download_init(fota_download_callback_t client_callback);
  * @param sec_tag_list Security tags that you want to use with HTTPS. Pass NULL to disable TLS.
  * @param sec_tag_count Number of TLS security tags in list. Pass 0 to disable TLS.
  * @param pdn_id Packet Data Network ID to use for the download, or 0 to use the default.
- * @param fragment_size Fragment size to be used for the download.
- *			If 0, @kconfig{CONFIG_DOWNLOAD_CLIENT_HTTP_FRAG_SIZE} is used.
+ * @param fragment_size Fragment size to be used for the download. If 0, no fragment is used.
```

Comment:
Align with the rest of the file:
```suggestion
 * @param fragment_size Fragment size to be used for the download. If 0, no fragmentation is used.
```